packages/perseus-core/src/widgets/core-widget-registry.ts

```diff
@@ -37,14 +37,14 @@ import type {
 } from "./logic-export.types";
 import type {Alignment} from "../types";
 
-const widgets = {};
+const widgets: Record<string, WidgetLogic> = {};
 
 function registerWidget(type: string, logic: WidgetLogic) {
     widgets[type] = logic;
 }
 
-export function isWidgetRegistered(type: string) {
-    const widgetLogic = widgets[type];
+export function isWidgetRegistered(type: string): boolean {
+    const widgetLogic: any = widgets[type];
```

Comment:
I don't know what's going on with this. ESLint thinks that widgetLogic can't be undefined (it can be) so it doesn't like that I try to do a type coercion on the next line.